README.md

```diff
@@ -6,11 +6,13 @@
 Unified Workflow tools for use with applications with UFS and beyond
 
 ## Installation
-Simple installation instructions
+Simple installation instructions with a conda environment.
 ```sh
 $> git clone https://github.com/ufs-community/workflow-tools
+$> conda env create -f environment.yaml
+$> conda activate workflow_tools
 $> cd workflow-tools
```

Comment:
Sorry for so many iterations! I think that the user will need to be in `workflow-tools` first, so this would need to come before the `conda env create` line.

Reply:
Thats quite alright, thanks for all of the time! Yeah, I just duplicated the read-the-docs text, let me add the `cd ./workflow-tools`

Reply:
Yeah, I'm just rushing, I need to take my time a little more, thanks